### DISCUSSION TITLE
[com_redirect]: the ability to select an Article, Category or Media

### DISCUSSION THREAD

**Discussion by 4digitalworld** [joomla/joomla-cms#32651] [1. Feature Evaluation]
### Is your feature request related to a problem? Please describe.
Now in the  `Redirects` component it is possible only to manually write the url-address for redirection, but the new address can also be changed in the future. 

### Describe the solution you'd like
I suggest adding a button to select `Article`, `Category` and `Media` for the `New URL` parameter. 

### Additional context
I think it might look something like this: 

![изображение](https://user-images.githubusercontent.com/3662519/108071894-4ff7ef80-7077-11eb-9f62-619472ec70ae.png)

**Comment by Blakelist7** [comment]
Hey, @infograf768 I am new to the community, mind guiding me to solve this issue?

**Comment by 4digitalworld** [comment]
This feature could fix a lot of problems in the new Joomla router.
